MFVNeutralino/python/Vertexer_cff.py

```diff
@@ -16,11 +17,29 @@
 mfvVerticesAux = mfvSelectedVertices.clone(vertex_aux_src = 'mfvVerticesAuxPresel', min_ntracks = 3)
 
 mfvVertexSequenceBare = cms.Sequence(
+    mfvGenParticles * 
     jmtRescaledTracks *
     mfvVertexTracks *
     mfvVertices
     )
 
+mfvVertexSequence = cms.Sequence(
+    mfvVertexSequenceBare *
+    mfvVerticesAuxTmp *
+    mfvSelectedVerticesTmp *
+    mfvVerticesToJets *
+    mfvVerticesToLeptons *
```

Comment:
To be sure @abbywarden  does this break anything for the non-displaced-lepton analysis? Maybe as much a question for @pkotamnives as for you.